lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -678,6 +679,7 @@ module Resources
             ],
             'name': [
               {
+                'id': 'CI-12457994-0',
                 'use': 'official',
```

Comment:
lines 91 and 388

Reply:
Updated: https://github.com/cerner/fhir.cerner.com/pull/540/commits/b5231d66e5756faf6981fa501b35e18fad66554a